docs/DeveloperGuide.md

```diff
@@ -308,6 +308,28 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
+
+**Use case: Delete debt**
+
+**MSS**
+
+1.  User requests to list debtors
+2.  PayMeLah shows a list of debtors
+3.  User requests to delete the debts of a specific debtor in the list
+4.  PayMeLah deletes the debts of that person
+5.  PayMeLah displays that the person’s debts are deleted
```

Comment:
Style guide says to use generic numbering for ordered lists. i.e., use `1.` for every item. If possible please help to change the default numberings also thanks.